src/Mvc/Mvc.ApiExplorer/test/EndpointMetadataApiDescriptionProviderTest.cs

```diff
@@ -653,6 +617,39 @@ public void HandleAcceptsMetadata()
                 });
         }
 
+        [Fact]
+        public void HandleAcceptsMetadataWithTypeParameter()
```

Comment:
Let's also add a test case for the super-weird scenario where an implicit from-body parameter is overridden. I think we have this already for attributes with the AddsRequestFormatFromMetadata() test, but not for the extension method.